src/Umbraco.Web/PublishedCache/ContentCacheAdapter.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.XPath;
+using Umbraco.Core;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Core.Xml;
+using Umbraco.Web.PublishedCache.NuCache;
+
+namespace Umbraco.Web.PublishedCache
+{
+    internal class ContentCacheAdapter : IPublishedContentCache, IPublishedContentCache2
```

Comment:
Doesn't need to implement both since IPublishedContentCache2 implements IPublishedContentCache?

Reply:
Changed as suggested